scripts/imageIDI.py

```diff
@@ -105,7 +108,7 @@ def main(args):
         lbl1 = 'XX'
         for p in ('XX', 'RR', 'I'):
             try:
-                img1 = utils.build_gridded_image(dataDict, size=NPIX_SIDE//2, res=0.5, pol=p, chan=toWork)
+                img1 = utils.build_gridded_image(dataDict, size=NPIX_SIDE//2, res=res, pol=p, uv_size=uv_size, chan=toWork)
```

Comment:
Sorry still working on reworking this and utils to make sense together while preserving the choice between aperture and image plane inputs. I'll double back to revert the uv variables to point to size instead to prevent outside conflicts.